### DISCUSSION TITLE
is there any automatic schema generation like apollo server?

### DISCUSSION THREAD

**Discussion by hexdecimal16** [Netflix/dgs-framework#950] [General]
I have develped a few graphql server using apollo, now I am trying to create my new server using netflix dgs. I want to know if it is possible to generate schema automatically while building ?

**Comment by srinivasankavitha** [comment]
No we don't support automatically generating the schema SDL. You do have the ability to dynamically construct the schema yourself, so it doesn't need to be static, but it is not automatically generated.
